qualtran/bloqs/arithmetic.py

```diff
@@ -74,7 +122,7 @@ class Square(Bloq):
 
     @property
     def signature(self):
-        return Signature.build(a=self.bitsize, result=2 * self.bitsize)
+        return Signature.build(a=self.bitsize, result=2 * self.bitsize + 2)
```

Comment:
can you explain the additional two bits and update the docstring

Reply:
This is wrong on my part, it should just be 2n.